### PR TITLE
Added my name to HTML file

### PR DESCRIPTION
**Pull request by sajclarke**
- The index.html file has been modified to include embedded css within the <head>
- The index.html file has been modified to include a 'h2' title within the 'body' tag

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,12 @@
 <html>
+    <head>
+        <style>
```
Comment (dariothornhill) [inline index.html]: Hey buddy can you please put this css in an external file and resumbit your pr.

File: index.html [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+            h2{
+                color:'red'
```
Comment (dariothornhill) [inline index.html]: Please end your css lines with a semi colon even if it is the last line because beginners are watching!

**Review by dariothornhill — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (sajclarke) [reply index.html]: Sure thing

File: index.html (hunk 1 above)
Comment (sajclarke) [reply index.html]: Sure thing

**Review by dariothornhill — APPROVED** [review]

**Pull request merged**